.github/workflows/greengage-release.yml

```diff
@@ -0,0 +1,30 @@
+# Release creation workflow
+name: Greengage release
+
+on:
+  release:
+    types: [published]
+
+concurrency:
+  group: Greengage CI-${{ github.ref }}
+  cancel-in-progress: false
+
+jobs:
+  upload-to-release:
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+        - version: 6
+          extensions: deb ddeb
+          artifact_name: deb-packages
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
```

Comment:
Do composite actions use permissions? Should we also add permissions for actions here?

According [github documentation](https://docs.github.com/en/actions/reference/workflows-and-actions/workflow-syntax#permissions): `If you specify the access for any of these permissions, all of those that are not specified are set to none`.